server/index.js

```diff
@@ -1,104 +1,163 @@
-'use strict'
-const path = require('path')
-
-const express = require('express')
-const csp = require('helmet-csp')
-
-const {middleware: cache} = require('./cache')
-const {getMeta} = require('./list')
-const {allMiddleware, requireWithFallback} = require('./utils')
-const userInfo = requireWithFallback('./routes/userInfo') // AdHoc: Changed to requireWithFallback
-const pages = requireWithFallback('./routes/pages') // AdHoc: Changed to requireWithFallback
-const categories = requireWithFallback('./routes/categories') // AdHoc: Changed to requireWithFallback
-const playlists = requireWithFallback('./routes/playlists') // AdHoc: Changed to requireWithFallback
-const readingHistory = requireWithFallback('./routes/readingHistory') // AdHoc: Changed to requireWithFallback
-const redirects = requireWithFallback('./routes/redirects') // AdHoc: Changed to requireWithFallback
-const errorPages = requireWithFallback('./routes/errors') // AdHoc: Changed to requireWithFallback
-
-const userAuth = requireWithFallback('userAuth')
-const customCsp = requireWithFallback('csp')
-
-const app = express()
-// AdHoc: Added Body Parser to support posts
-app.use(express.json())
-
-const {preload, postload} = allMiddleware
-
-// The trust proxy flag tells the app to use https for links
-// and redirect urls if it sees indications that the request
-// passed through a proxy and was originally sent using https
-if ((process.env.TRUST_PROXY || '').toUpperCase() === 'TRUE') {
-  app.enable('trust proxy')
-}
+const log = require('./logger');
```

Comment:
another idea for code cleanliness 

what if you revert all the changes to `server/index.js`, and make the new entrypoint something like `server/bootstrap.js`, which... does all the secrets loading, and then loads index.js?

that way we don't have to have this massive diff for index.js

Reply:
I think that's a good idea.